client/blocks/conversation-follow-button/button.jsx

```diff
@@ -41,8 +41,8 @@ class ConversationFollowButton extends React.Component {
 		];
 		const iconSize = 20;
 		const label = isFollowing
-			? translate( 'Following Conversation' )
-			: translate( 'Follow Conversation' );
+			? translate( 'Following conversation' )
+			: translate( 'Follow conversation' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.